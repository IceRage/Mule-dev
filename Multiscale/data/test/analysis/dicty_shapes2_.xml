<?xml version="1.0" encoding="utf-8"?>
<!--Warning! This xml file was automatically generated by a C++ program using the Boost PropertyTree library.-->
<experiment>
	<timepoint>
		<spatialEntity spatialType="cluster">
			<clusteredness>0.002332466079056377</clusteredness>
			<density>3</density>
			<area>164641.3333333333</area>
			<perimeter>2806</perimeter>
			<distanceFromOrigin>796.6454669424787</distanceFromOrigin>
			<angle>11.659292653523</angle>
			<triangleMeasure>0.4974358974358974</triangleMeasure>
			<rectangleMeasure>0.9948717948717948</rectangleMeasure>
			<circleMeasure>0.1219320218563705</circleMeasure>
			<centroidX>703.5</centroidX>
			<centroidY>129</centroidY>
		</spatialEntity>
		<spatialEntity spatialType="cluster">
			<clusteredness>0.003481017489249063</clusteredness>
			<density>3</density>
			<area>316284.6666666667</area>
			<perimeter>2144.854042053223</perimeter>
			<distanceFromOrigin>72.56031973468694</distanceFromOrigin>
			<angle>93.35166834495223</angle>
			<triangleMeasure>0.5695785070785071</triangleMeasure>
			<rectangleMeasure>0.7296241681940597</rectangleMeasure>
			<circleMeasure>0.7415889895035143</circleMeasure>
			<centroidX>391.5481</centroidX>
			<centroidY>960.1702</centroidY>
		</spatialEntity>
		<spatialEntity spatialType="cluster">
			<clusteredness>0.00352995433403631</clusteredness>
			<density>3</density>
			<area>238296.6666666665</area>
			<perimeter>2287.053100585938</perimeter>
			<distanceFromOrigin>524.0772843770277</distanceFromOrigin>
			<angle>50.48509434964318</angle>
			<triangleMeasure>0.9082392676686462</triangleMeasure>
			<rectangleMeasure>0.5497168390503185</rectangleMeasure>
			<circleMeasure>0.329670161036458</circleMeasure>
			<centroidX>1096.131</centroidX>
			<centroidY>1057.571</centroidY>
		</spatialEntity>
		<numericStateVariable>
			<name>avgClusterednessClusters</name>
			<value>0.5785832846798725</value>
		</numericStateVariable>
		<numericStateVariable>
			<name>avgDensityClusters</name>
			<value>3</value>
		</numericStateVariable>
	</timepoint>
</experiment>
