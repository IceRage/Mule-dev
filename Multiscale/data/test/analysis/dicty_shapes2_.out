Clusteredness degree,Density,Area,Perimeter,Distance from origin,Angle(degrees),Shape,Triangle measure,Rectangle measure,Circle measure,Centre (x-coord),Centre (y-coord)
0.00233247,3,164641,2806,796.645,11.6593,rectangular,0.497436,0.994872,0.121932,703.5,129
0.00348102,3,316285,2144.85,72.5603,93.3517,circular,0.569579,0.729624,0.741589,391.548,960.17
0.00352995,3,238297,2287.05,524.077,50.4851,triangular,0.908239,0.549717,0.32967,1096.13,1057.57

Average clusteredness degree: 0.578583
Average density: 3
